src/commands/base.ts

```diff
@@ -114,10 +133,15 @@ export default abstract class Base extends Command {
             this.configPath = flags['config-path']
         }
 
-        this.configFromFile = this.loadConfig(this.configPath)
+        if (flags['repo-config-path']) {
+            this.repoPath = flags['repo-config-path']
```

Comment:
it's probably worth updating the property name to match, just for clarity
```suggestion
            this.repoConfigPath = flags['repo-config-path']
```